src/promptflow/promptflow/_cli/_pf/_experiment.py

```diff
@@ -203,27 +200,31 @@ def dispatch_experiment_commands(args: argparse.Namespace):
 
 
 def create_experiment(args: argparse.Namespace):
-    from promptflow._sdk._load_functions import _load_experiment_template
-
-    template_path = args.template
-    logger.debug("Loading experiment template from %s", template_path)
-    template = _load_experiment_template(source=template_path)
-    logger.debug("Creating experiment from template %s", template.dir_name)
-    experiment = Experiment.from_template(template, name=args.name)
-    logger.debug("Creating experiment %s", experiment.name)
-    exp = _get_pf_client()._experiments.create_or_update(experiment)
-    print(json.dumps(exp._to_dict(), indent=4))
+    if not Path(args.template).is_absolute():
+        raise UserErrorException("Please provide the absolute path to the experiment template.")
+    if not Path(args.template).exists():
+        raise UserErrorException(f"Experiment template path {args.template} does not exist.")
+    experiment_name = args.name
+    timestamp = datetime.datetime.now().strftime("%Y%m%d_%H%M%S_%f")
+    experiment_name = experiment_name or f"{Path(args.template).parent.name}_{timestamp}"
+    logger.debug("Creating experiment %s", experiment_name)
+    exp = _get_pf_client()._pfs_client.create_experiment(name=experiment_name, template=args.template)
```

Comment:
add comments somewhere on why we call PFS api not directly call operations here.

Reply:
I think we can still avoid PFS API for list/show apis?